### PR TITLE
KOGITO-555 About page modifications

### PR DESCRIPTION
**Pull request by AjayJagan**
- added modifications to about page

### CODE REVIEW

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
   AboutModal,
-  Button,
   TextContent,
+  Text,
   TextList,
   TextListItem
 } from '@patternfly/react-core';
 import './AboutModal.css';
+import kogito_logo_aboutpage from '../../../static/kogito_logo_aboutpage.svg';
+import KogitoAbout from '../../../static/KogitoAbout.png';
```
Comment (srambach) [inline packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: Should we establish a better convention for image naming and the import names of them?

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx [hunk 2]
```diff
@@ -20,29 +23,55 @@ const AboutModalBox: React.FC<IOwnProps> = ({
...
           <TextListItem component="dd">
-            EvmRole-super_administrator
+            <a
+              href="https://github.com/kiegroup/kogito-runtimes/blob/master/LICENSE"
+              target="_blank"
+            >
+              https://github.com/kiegroup/kogito-runtimes/blob/master/LICENSE
+            </a>
+          </TextListItem>
+          <TextListItem component="dt">Report a bug : </TextListItem>
```
Comment (srambach) [inline packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: I don't think we need the extra space before the colon (applies to all the `TextListItem`s

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx [hunk 3]
```diff
@@ -20,29 +23,55 @@ const AboutModalBox: React.FC<IOwnProps> = ({
...
-          <TextListItem component="dt">Cloudforms Version</TextListItem>
-          <TextListItem component="dd">4.1</TextListItem>
-          <TextListItem component="dt">Server Name</TextListItem>
-          <TextListItem component="dd">40DemoMaster</TextListItem>
-          <TextListItem component="dt">User Name</TextListItem>
-          <TextListItem component="dd">Administrator</TextListItem>
-          <TextListItem component="dt">User Role</TextListItem>
+          <TextListItem component="dt">Version : </TextListItem>
+          <TextListItem component="dd">{version}</TextListItem>
+          <TextListItem component="dt">License Information : </TextListItem>
```
Comment (srambach) [inline packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: lower case "I" - "License information:"

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx [hunk 4]
```diff
@@ -20,29 +23,55 @@ const AboutModalBox: React.FC<IOwnProps> = ({
     <AboutModal
       isOpen={isOpenProp}
       onClose={handleModalToggleProp}
-      trademark="Trademark and copyright information here"
+      trademark="Management Console is open source software, released under the Apache Software License 2.0"
```
Comment (srambach) [inline packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: Should this say Management Console or Kogito? Or just avoid that decision and say "This open source software is released under the Apache Software License 2.0" ? @cristianonicolai do you know? Another possibility is to move this to the License Information line, put the link words as "Apache Software License 2.0" and then link to the license url.

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 1 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: I will follow 'camelCase' sorry :)

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 4 above)
Comment (cristianonicolai) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: @srambach  perhaps we could say: _Management Console is part of Kogito, a open source software..._

**Comment by cristianonicolai** [comment]
@AjayJagan can you update the commit message and PR description to include the JIRA number, please.

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 2 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: done :+1:

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 3 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: done :+1:

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 4 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: I have added -Management Console is part of Kogito, an open source software released under the Apache Software License 2.0. wdyt?

**Comment by AjayJagan** [comment]
> @AjayJagan can you update the commit message and PR description to include the JIRA number, please.

done :+1:

File: packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx (hunk 4 above)
Comment (srambach) [reply packages/management-console/src/components/Molecules/AboutModalComponent/AboutModal.tsx]: 👍 sounds good to me

**Review by srambach — APPROVED** [review]
Looks good to me!

bitmoji

**Review by cristianonicolai — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Pull request merged**